templates/nextjs/src/sway-api/contract-ids.json

```diff
@@ -1,3 +1,3 @@
 {
-  "testContract": "dummy-contract-id"
-}
+  "testContract": "0x5cfe9ecc6b380e45e5b5f59e9badbd05c095c330c7d2bcb8da5a2f6bfe088c53"
```

Comment:
Shouldn't this be generated for each user?

Reply:
oops https://github.com/FuelLabs/fuels-ts/pull/2798/commits/c5612888786f357e1c1890b5a969f8c2bb8bd33c